pkg/build/actions/bump-version/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"path/filepath"
+	"strings"
+
+	"dagger.io/dagger"
+)
+
+func main() {
+	var (
+		dir     = flag.String("dir", ".", "The grafana directory")
+		version = flag.String("version", "", "Use the '-version' flag to manually set the version")
+		ctx     = context.Background()
+	)
+	flag.Parse()
+	if *version == "" {
+		log.Fatalln("-version must be set")
+	}
+
+	d, err := dagger.Connect(ctx)
+	if err != nil {
+		panic(err)
+	}
+
+	src := d.Host().Directory(filepath.Clean(*dir), dagger.HostDirectoryOpts{
+		Include: []string{
+			".nvmrc",
+			".yarn",
+			".yarnrc.yml",
+			"nx.json",
+			"project.json",
+			"lerna.json",
```

Comment:
aren't we killing `lerna` for `nx`?

Reply:
probably but i'm just copying what the existing behavior is. If anything I think we'll continue to have lerna call nx.